### PR TITLE
Documentation changes for V3.5  Library gate

### PR DESCRIPTION
**Pull request by mbaillif**
Many files are modified with the correction of the "Lang" tag.
But the main changes concern the gates library doc.

### CODE REVIEW

**Comment by BFH-ktt1** [comment]
@mbaillif : When a PR is ready for merge could you please assign a reviewer? After the reviewer say's ok we can merge the PR.

**Review by BFH-ktt1 — DISMISSED** [review]

File: src/main/resources/doc/en/html/libs/gates/buffer.html [hunk 1]
```diff
@@ -173,7 +173,7 @@ <h2>
           East edge :
         </dt>
         <dd>
-          The output, which always matches the input into the left side. <b class="propertie">Bits attribute</b>.
+          The output, which always matches the input into the left side. Bit width according to Data  <b class="propertie">Bits attribute</b>.
```
Comment (MarcinOrlowski) [inline src/main/resources/doc/en/html/libs/gates/buffer.html:176]: That's bit odd looking. Shouldn't the "Data Bits" be of class "properties", not just "Bits"?

File: src/main/resources/doc/en/html/libs/gates/controlled.html [hunk 2]
```diff
@@ -74,86 +74,203 @@ <h2>
...
       </p>
       <h2>
         Pins (assuming component faces east, control line right-handed)
       </h2>
+	  <div class="attliste">
       <dl>
-        <dt>
-          West edge (input, bit width matches Data Bits attribute)
+        <dt class="lib">
+          West edge :
```
Comment (MarcinOrlowski) [inline src/main/resources/doc/en/html/libs/gates/controlled.html:209]: No space before punctuation chars, please (**multiple** places).

File: src/main/resources/doc/libstatus.html [hunk 3]
```diff
@@ -380,32 +380,32 @@ <h1 id=lib>Library Status Dt:08.08.2021  In progress</h1>
...
 	<tr>
-		<td height="17" align="center"></td>
+		<td height="17" bgcolor="#72BF44">3.5</td>
 		<td align="left"></td>
 		<td align="left"></td>
 		<td align="left">XOR/XNOR/Even/Odd Gate</td>
 		<td align="left"></td>
-		<td align="center" bgcolor="#C2E0AE"><a href="./en/html/libs/gates/xor.html">__</a></td>
-		<td align="center" bgcolor="#C2E0AE"><a href="./fr/html/libs/gates/xor.html">__</a></td>
+		<td align="center" bgcolor="#72BF44"><a href="./en/html/libs/gates/xor.html">__</a></td>
```
Comment (MarcinOrlowski) [inline src/main/resources/doc/libstatus.html:388]: Can we have colors moved to CSS?

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]

**Comment by MarcinOrlowski** [comment]
I am voiting to block PRs (or automatically rejects them) if too many files are affected. 

If you want to add language to HTML, i.e.:

    <html lang="pt">

then because of the number of files it should go as separate PR.

**Comment by mbaillif** [comment]
Je vais reprendre tout cela.
Thanks for the comments.

**Comment by maehne** [comment]
@MarcinOrlowski: I think we should show some tolerance when reviewing PRs from other contributors -- especially if they do their first contributions to the project or, as is the case with @mbaillif, have been continuously maintaining/improving a neglected part of this project, i.e., in this case the documentation. @mbaillif has been contributing improvements to the documentation for several years now, which helped considerable to improve its quality. We should be careful to not frustrate our contributors. Reviewing PRs and pointing out issues, which should be improved is fine as is making suggestions on how to make our and their work easier, e.g., by making topical commits or limiting the scope and size of a PR. Nevertheless, there may be times when it makes sense to propose many changes in one go to consistently rework some aspect of the project. In these cases, we should remind the contributor that it helps if they explain their modifications in the commit messages and/or PR message.

@mbaillif: How would you like to proceed? Can you split the PR with reasonable effort into smaller chunks or do you need someone to review this larger PR? In case of the latter, feel free to reopen this PR and I will try to have a look in the coming days.

**Pull request closed without merging**